pkg/controller/credentialsrequest/credentialsrequest_controller.go

```diff
@@ -284,6 +290,22 @@ func (r *ReconcileCredentialsRequest) Reconcile(request reconcile.Request) (reco
 	origCR := cr
 	cr = cr.DeepCopy()
 
+	// Ignore CR if it's for a different cloud
+	differentCloud, err := isCRForDifferentCloud(cr, r.platformType)
```

Comment:
flipping the boolean response here might be a little more readable, something like "matchesPlatform" returning true.

Reply:
done